sbpy/activity/tests/test_dust.py

```diff
@@ -1,19 +1,46 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
 import os
+import sys
+import mock
 import pytest
 import numpy as np
 import astropy.units as u
 from astropy.tests.helper import remote_data
 from astropy.utils.data import get_pkg_data_filename
 import synphot
 from ..dust import *
```

Comment:
This could be replaced by `from ..dust import Afrho, Efrho, phase_HalleyMarcus` for clarity